### ISSUE TITLE
[Accessibility][Keyboard - Create Node]: Keyboard focus is not visible on 'Create Node' button.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2061]
**User Experience:**
Keyboard dependent users would face difficulty if keyboard focus does not visible on the control.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and set the zoom at 400%.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the flow chart.
4. Navigate to "+" button available in the flow chart and observe the issue.

**Actual:** 
The keyboard focus is not visible at "+" button i.e. create node button.

**Expected:**
The focus should be clearly visible to each and every control available at the page.

**MAS Guideline:** [MAS2.4.7](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c})

**Attachment:**
[MAS2.4.7_Focus not visible on Add node button.zip](https://github.com/microsoft/BotFramework-Composer/files/4241294/MAS2.4.7_Focus.not.visible.on.Add.node.button.zip)

**Comment by corinagum** [comment]
@DesignPolice Do specs already have focus for this element? If not, this may need designer input before fix. :)
Feel free to unassign yourself if no input needed.

**Comment by DesignPolice** [comment]
Brought this one up in Stand Up this morning, it a relatively minimal change, but I think it brings up a bigger focus issue in the Visual Editor. 

Adding @mareekuh @a-b-r-o-w-n (they were in the stand-up)

We have Focus on items in the Visual Editor as we "tab" through items the state is a blue outline. When the blue outline is on white we hit our 4.5:1 number - with the background of F6F6F6 - it knocks us down to 4.19:1 - not passing. 

So my first thought was okay we can just use a darker blue, but then looking at some of the other nodes we have some inconsistent things going on.

<img width="954" alt="Screen Shot 2020-02-27 at 12 29 01 PM" src="https://user-images.githubusercontent.com/45223620/75484118-02adda00-595d-11ea-8455-4d830b9fc7a1.png">

<img width="1022" alt="Screen Shot 2020-02-27 at 12 28 55 PM" src="https://user-images.githubusercontent.com/45223620/75484123-03df0700-595d-11ea-996a-f117d883232a.png">

So I need to get some clarity on what we want to do - 

The Blue will need to be darker if we are going to use it - #0078D4 would need to be #004E8C

Black is fine for the other state... 

for Focus on the "+" I would like it to be a circle like this, but probably should get some consensus on that

![Screen Shot 2020-02-27 at 12 37 29 PM](https://user-images.githubusercontent.com/45223620/75484641-055cff00-595e-11ea-934b-799f060bdf08.png)

![Screen Shot 2020-02-27 at 12 37 43 PM](https://user-images.githubusercontent.com/45223620/75484639-042bd200-595e-11ea-8078-e729afa0ba10.png)

PVA does a mix of really heavy drop shadow and a border around the "+" not a huge fan, but that might be something that Lili thinks we should align. ( PVA Below )

<img width="344" alt="Screen Shot 2020-02-27 at 12 51 36 PM" src="https://user-images.githubusercontent.com/45223620/75485769-2292cd00-5960-11ea-8a4d-4b38ea9ff255.png">


Let the comments begin. 

thanks Marc

**Comment by mareekuh** [comment]
Tab = black outline
Selection = blue outline
For the +, the outline should be around the circle, not a square.

________________________________

**Comment by DesignPolice** [comment]
Okay that works. @corinagum do we need to alter the blue color to hit contrast? It does fall a little short on the #F6F6F6 background.

**Comment by corinagum** [comment]
I would say play it safe and pick the darker color, if that's alright with you guys.

**Comment by DesignPolice** [comment]
okay... then go with this. That would affect anything on the Visual Canvas then... so all of the nodes.

The Blue will need to be darker if we are going to use it - #0078D4 would need to be #004E8C

**Comment by yeze322** [comment]
@alanlong9278 let's solve it along with other 'tab navigation' issues

**Comment by ashish315** [comment]
As checked the focus is now visible clearly on the node. But the pane is now not accessible with Tab key. On pressing tab from 'Show Code' the focus should come inside the tree pane but the focus stuck on the whole pane. As well as when we use mouse to take focus inside the pane the keyboard focus is not visible.
Please refer the video's for the same "[Issue.zip](https://github.com/microsoft/BotFramework-Composer/files/4524669/Issue.zip)"

I downloaded below Master build file last updated on 23rd April.

**Test Environment**
Browser: Microsoft Edge Dev {Version 84.0.488.1 (Official build) dev (64-bit)}
OS build: 2004 (19562.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
![Github](https://user-images.githubusercontent.com/60913131/80138432-6f8eca80-85c2-11ea-88d4-e617ae22ae4f.png)

**Comment by ashish315** [comment]
@corinagum If you want we can raise new issue for the same then we are ready to do the same.
